etl/src/process/trie.rs

```diff
@@ -0,0 +1,92 @@
+//! prefix tree
+
+use std::{collections::BTreeMap, ops::Deref};
+
+#[derive(Default)]
```

Comment:
Isn't there any library that has implemented Trie?

Reply:
Yes, but that's not what we needed.
What `trie_rs` does is **only** add count at leaf node, but we need it along path.
For example, we have `http://example.com/about/eason` and `http://example.com/about`, the user just see the fact he visitied `http://example.com/about` twice.